app/src/features/apiClient/screens/apiClient/components/clientView/APIClientView.tsx

```diff
@@ -375,6 +375,21 @@ const APIClientView: React.FC<Props> = ({ apiEntry, apiEntryDetails, notifyApiRe
     (evt.target as HTMLInputElement).blur();
   }, []);
 
+  const onUrlKeyDown = useCallback(
+    (evt: KeyboardEvent, text: string) => {
+      if (evt.metaKey) {
+        if (evt.key.toLowerCase() === "s") {
+          evt.preventDefault();
+          onSaveButtonClick();
+        } else if (evt.key.toLowerCase() === "enter") {
+          evt.preventDefault();
+          onSendButtonClick();
+        }
+      }
+    },
+    [onSaveButtonClick, onSendButtonClick]
+  );
+
```

Comment:
`preventDefault()` is not required here as this is already handled in SingleLineEditors onKeyDown listener